<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_delete</class>
 <widget class="QDialog" name="Dialog_delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>请输入待删除植株的名称:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>请输入日期:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEditCalendar" name="lineEdit_date"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btn_delete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_close">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEditCalendar</class>
   <extends>QLineEdit</extends>
   <header>./calendar/qlineeditcalendar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
